addons/crm/models/crm_lead.py

```diff
@@ -743,8 +777,23 @@ def create(self, vals_list):
         for vals in vals_list:
             if vals.get('website'):
                 vals['website'] = self.env['res.partner']._clean_website(vals['website'])
+        
         leads = super().create(vals_list)
 
+        missing_partner_leads = leads.filtered(
```

Comment:
is there a context key for quick create that we could use to only filter records when needed?